control-services/images/jenkins/casc_template.yaml

```diff
@@ -2,7 +2,7 @@ jenkins:
   authorizationStrategy:
     globalMatrix:
       permissions:
-      - Overall/Administer:admin
+      - Overall/Administer:${JENKINS_ADMIN_ID}
```

Comment:
Can you point me to the reference for this? Struggling to find online, wanted to read up on it

Reply:
If you go to this [tutorial](https://www.digitalocean.com/community/tutorials/how-to-automate-jenkins-setup-with-docker-and-jenkins-configuration-as-code#step-5-%E2%80%94-setting-up-authorizatione), which I believe is the one you followed to set up the original yaml configuration, it specifies in step 5 that the format for this permission is `<permission-group>/<permission-name>:<role>`, where `<role>` is the name of the user we are granting access to

Reply:
Right you are....that is the article I posted and read (mostly)...Thanks for the sanity check, I definitely thought that did something different earlier.